test/README

```diff
@@ -9,3 +9,12 @@ symlinked from test/output/latest/html/index.html.
 
 *--output-dir*
 	Litani will write all of its test output files for this run to this path
+
+*--fast*
+    Do not run tests with SLOW variable set to True
+
+## Writing Tests
+
+Each test is made up of 4 jobs: init, add-jobs, run-build, and check-run
+If --fast flag is passed, the jobs are given the a timeout=10s
```

Comment:
>  If --fast flag is passed, the jobs are given the a timeout=10s

I suggest: ` If --fast flag is passed, each job is given a timeout of 10 seconds. In the worst case, a test would take 40 seconds to run.`